test/suites/common/upgrade-crds.yaml

```diff
@@ -15,5 +17,6 @@ spec:
   - name: upgrade-crds
     image: public.ecr.aws/karpenter-testing/tools:latest
     script: |
+      aws eks update-kubeconfig --name $(params.cluster-name)
```

Comment:
idr if the region is already set at this point, did you try this out within tekton?

Reply:
Yes, tested this against a cluster. This is the same call that occurs in the `setup.yaml`